match/lib/match.rb

```diff
@@ -29,4 +29,21 @@ def self.environments
   def self.enterprise?
     ENV["MATCH_FORCE_ENTERPRISE"]
   end
+
+  def self.type_is_enterprise?(type)
+    Match.enterprise? && type != "development"
```

Comment:
So running _match_ with type `adhoc` or `appstore` will both use the `enterprise` type? Is that by design?